### PR TITLE
Vp2410/rel v1.1.0

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

**Comment by miczyg1** [comment]
> @miczyg1 can you fix or comment on the CI errors?

One copy-pasted paragraph in openness_score.md which is obsolete, removed: 756d3da7889f5d0152fcdf7c38fc5b8cd3899c2b

Broken link to GLK FSP (bug in templates, fixed in the templates), updated: be56759560516eb297cc36cad3ca00a14d4cd9a3

For the rest of broken links, I have no idea, didn't touch them.

**Review by mkopec — APPROVED** [review]

**Comment by mkopec** [comment]
Links appear to all be working now

File: docs/variants/protectli_vp2410/releases.md [hunk 1]
```diff
@@ -14,6 +14,71 @@ For details about our release process please read
 Test results for this platform can be found
 [here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=975611333).
 
+## v1.1.0 - 2024-04-09
```
Comment (BeataZdunczyk) [inline docs/variants/protectli_vp2410/releases.md:17]:
```suggestion
## v1.1.0 - 2024-05-16
```

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]

**Comment by mkopec** [comment]
Rebased, updated the release date and updated the openness score pngs. @BeataZdunczyk

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]
@mkopec Link to the Test Results needs updating to https://docs.google.com/spreadsheets/d/1wSE6xA3K3nXewwLn5lV39_2wZL1kg5AkGb4mvmG3bwE/edit#gid=1033426620

**Comment by mkopec** [comment]
@BeataZdunczyk fixed

**Pull request merged**